### ISSUE TITLE
[20231213]: Windows DNS integration

### ISSUE THREAD

**Issue by gcoon151** [coreybutler/node-windows#363]
**Is your feature request related to a problem? Please describe.**
It's not a problem per se, but I can put it in problem statement: My code running in node-windows needs to expose to the network with a DNS name.

**Describe the solution you'd like**

Windows has DNS services built into it. Ideally I'd just configure node-windows to say update $DNS-FQDN on windows DNS for this instance IP. The ultimate would be when I have multiple instances running on multiple windows servers DNS would provide the IPs of all the instances (at least up to 9 IPs)

Basically integrating something like this https://learn.microsoft.com/en-us/answers/questions/501319/need-a-script-for-updating-dns-host-record
into node-windows config file. 

**Describe alternatives you've considered**
Manually configuring all the DNS for this. 

**Additional context**

<table>
  <tr>
    <td>Please use the reactions to cast a vote in favor of or against this feature suggestion &raquo;</td>
    <td><img src="https://github.com/metadoc/metadoc.io/raw/main/.github/ISSUE_TEMPLATE/vote.png" width="75"/></td>
  </tr>
</table>

**Comment by coreybutler** [comment]
This is beyond the scope of what this project is intended for. It's possible to do this "manually" using Node's `exec` function, so that's the workaround you could use now.

Dynamic/toggled DNS servers are something on our radar for a different product. Unfortunately, that product isn't scheduled to start until later next year at the earliest. The plan, at this time, is to implement something like this in an add-on to [Runtime](https://github.com/coreybutler/nvm-windows/wiki/Runtime).

**Comment by gcoon151** [comment]
Appreciate the consideration and suggestion. Best of luck and I'll follow the runtime project repo.
